CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to the Zowe Launcher package will be documented in this file
 This repo is part of the app-server Zowe Component, and the change logs here may appear on Zowe.org in that section.
 
 ## 2.13.0
+- Bugfix: Changed timestamp to UTC to match the server timestamps (#103)
+- Bugfix: Removed server timestamps from syslog to avoid duplicate logging of time (#103)
+- Bugfix: Avoided hang on components when components were logging messages longer than 1024 characters. (#103)
+- Enhancement: syslog output per line is capped at 512 bytes, extra characters will be omitted (#103)
+- Enhancement: Added milliseconds logging to match the server timestamps (#103)
```

Comment:
But where is this added? I am confused. You created

`struct timeval now;
  gettimeofday(&now, NULL);
  int milli = now.tv_usec / 1000;
  snprintf(result.value+strlen(result.value), 5, ".%03d", milli);`
  
  1. but milli isn't being logged anywhere I can see
  2. Also, milli is based on gettimeofday so you should probably add a comment here that to use a reliable milliseconds, you need to call `gettimeofday` every time you use milli in a log message

Reply:
1. it is, in the job log. you wouldnt see it in the syslog because it gets omitted on purpose.
2. thats not relevant information to end users (changelog). thats an implementation detail.

Reply:
its right here 
![image](https://github.com/zowe/launcher/assets/30730276/db1cde80-7a56-4359-917a-3e6e45b863bf)

Reply:
Yes, I meant add a code comment @1000TurquoisePogs to 2)